src/benchmark/ips.cr

```diff
@@ -65,7 +65,7 @@ module Benchmark
         @items.each do |item|
           GC.collect
 
-          count = 0
+          count = 0_i64
```

Comment:
Suggestion: `UInt64` would be even safer.

```suggestion
          count = 0_u64
```

Reply:
I originally did use an unsigned integer but I ended up swapping to `Int64` for consistency with how the counters in `#run_calculation` are typed.

https://github.com/crystal-lang/crystal/blob/360eda983b3ced372903f29443c546997eba5531/src/benchmark/ips.cr#L87-L88

Considering the timescales needed to even approach the signed 64 bit integer limit consistency and style is probably more important.

Reply:
Still, there's no downside to using an unsigned integer... so why not?